app/controllers/guides_controller.rb

```diff
@@ -11,6 +12,8 @@ def index
 
   def show
     @guide = Guide.friendly.find(params[:id])
+    add_breadcrumb @guide.source_set.name, source_set_path(@guide.source_set)
+    add_breadcrumb @guide.name, guide_path(@guide)
```

Comment:
Like my source comment (below), could we have the text of the link be "Teaching guide" instead of `@guide.name`?  Again, because of the length of the names.